package.json

```diff
@@ -59,6 +59,7 @@
     "gfm.css": "^1.1.2",
     "highlight.js": "^10.1.2",
     "jsrsasign": "^9.1.5",
+    "katex": "^0.12.0",
```

Comment:
won't this cause further issues with dependency version mismatch?

Reply:
It does mean it's a dependency in two layers, which adds some complexity, but it's also a much more correct expression of what we're actually doing. Also, without this PR, we're relying on a specific pattern layout of nested `node_modules` directories, which is undefined behaviour, and seemed to be breaking the RC build today, so I think that's enough evidence that we need this change.

Reply:
yea, understandable. Just concerned that we're going to forget to update the dependency in both layers.